EpsilonCCS/Src/TelemetryReporting/TelemetryReporting.c

```diff
@@ -201,7 +201,7 @@ void sendBatteryFaults()
 
     packetPayload[0] = BATTERY_FAULTS_PKG_ID;
     writeBoolsIntoArray(packetPayload, 1, &batteryFaultsData.batteryErrorFlags, 21);
-    writeBoolsIntoArray(packetPayload, 5, &batteryFaultsData.batteryLimitFlags, 14);
+    writeBoolsIntoArray(packetPayload, 4, &batteryFaultsData.batteryLimitFlags, 16);
     addChecksum(packetPayload, BATTERY_FAULTS_LENGTH);
```

Comment:
Is the length of 7 still accurate? otherwise looks ok to me

Reply:
I think we changed it to 6

Reply:
Good catch! Yes, please update this to 6.